packages/@tinacms/cli/src/cmds/forestry-migrate/util/index.ts

```diff
@@ -15,9 +15,34 @@ import fs from 'fs-extra'
 import path from 'path'
 import yaml from 'js-yaml'
 import z from 'zod'
-import type { TinaFieldInner } from '@tinacms/schema-tools'
+import type { TinaFieldInner, TinaTemplate } from '@tinacms/schema-tools'
 import { logger } from '../../../logger'
+import { warnText } from '../../../utils/theme'
+import { ErrorSingleton } from './errorSingleton'
 
+const errorSingletonInstance = ErrorSingleton.getInstance()
+
+export const stringifyName = (name: string, template: string) => {
```

Comment:
Are we changing the name of their fields in their content pages along with this? I have a bit of concern it might be some overhead for the user to update their site to make sure it doesn't break the build (as opposed to, if we were to just support characters like dashes in our field names)